src/main/java/com/solvd/sql/model/Order.java

```diff
@@ -21,16 +21,16 @@ public class Orders {
     @XmlElement(name = "orderDate")
     @XmlJavaTypeAdapter(DateAdapter.class)
     @JsonProperty("orderDate")
-    @JsonFormat(shape= JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
     private Date orderDate;
 
     @XmlElement(name = "total")
     @JsonProperty("total")
     private double total;
 
-    @XmlElement(name = "customerId")
-    @JsonProperty("customerId")
-    private int customerId;
+    @XmlElement(name = "customer", type = Customer.class)
+    @JsonProperty("customer")
+    private Customer customer;
 
```

Comment:
Can the order have a list or collection of order items?

Reply:
I will implement it on the next PR